### PR TITLE
Added contact form

### PR DESCRIPTION
**Pull request by wajahatalli**
# Changes in this Pull Request

- Added h1 tag in the Contact form
- Added description in the Contact form
- Added the contact form 
- Added styles to it, to make it look like a Figma template
- Added a way to receive the Forms submitted by others
- changed the type="button" to type="submit"
- used one font style across the whole HTML
- Made required changes in COntact Form design.

### CODE REVIEW

**Review by anagudelogu — CHANGES_REQUESTED** [review]
Hi team! @wajahatalli @Ellon-M, 

Great job working on your contact section! 🍾 🎆 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
✅ Professional and descriptive PR title and summary. 💯 
✅ Great use of GitHub flow. 👍🏼 
✅ Meaningful and descriptive commit messages. 🥇 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by wajahatalli** [comment]
1. Added a way to receive the Forms submitted by others
2. changed the type="button" to type="submit"
3. used one font style across the whole HTML
4. Made required changes in Contact Form design.

File: index.html [hunk 1]
```diff
@@ -1,257 +1,272 @@
...
+        <input type="email" name="_replyTo" class="inp-form" id="email" placeholder="Email" required />
+        <label for="message"></label>
+        <textarea name="message" class="text-area" cols="30" rows="10" maxlength="500"
+          placeholder="Write your message here" required id="message"></textarea>
+        <small id="message-success" class="hidden">From Submitted</small>
+        <input type="submit" class="btncontact" value="Get It Touch" />
+      </form>
+      <div class="contact-bar">
+        <img class="end-line" src="assets/images/Shape.png" alt=" end line icon" />
+      </div>
```
Comment (Alfredbis29) [inline index.html:266]: - [ OPTIONAL] I can see that you are using `img` tag to put line bar, on my opinion this cannot be an `img` tag, I can suggest you to use a `div` tag then give it a class name and design it in your style file  :wink:

**Review by Alfredbis29 — APPROVED** [review]
# STATUS: APPROVED  
 ## Hi @wajahatalli ,
Your project is complete! There is nothing else to say other than... Great job clap Congratulations! :tada:
 
 TO HIGHLIGHT:
 
 * Your project is professional :white_check_mark:
 * Your repo is professional :white_check_mark:
 * Well structured files :white_check_mark:
 * No Linter errors :white_check_mark:
 
 Now you can merge this branch and move on :1st_place_medal: HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers four_leaf_clover:


 
 ## Optional suggestions

- [ ] Check below the review 

 _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
 
Cheers and Happy coding! :clap:  :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (wajahatalli) [reply index.html:266]: Okay. thank you, i'll consider.

**Pull request merged**